pym/bob/scm/git.py

```diff
@@ -278,14 +281,17 @@ def status(self, workspacePath, dir, verbose=0):
             ret = 'clean'
             if status == "":
                 if verbose >= 3:
-                    print(colorize("   STATUS      {0}".format(scmdir), "32"))
+                    print(colorize("   STATUS {0}     {1}".format(overriddenStatus, scmdir), "32"))
             elif status != "u":
                 ret = 'unclean'
 
             if (status != "") and (verbose != 0):
-                print(colorize("   STATUS {0: <4} {1}".format(status, scmdir), "33"))
+                print(colorize("   STATUS {0}{1: <4} {2}".format(overriddenStatus, status, scmdir), "33"))
```

Comment:
Minor: that will change the indentation if an override is active.

But more importantly I'm thinking about the expected behavior of here. Unless I use `-vv` it will depend on the status of the working tree whether I see that an override is active or not. I could imagine that this may surprise users..?